### PR TITLE
Adding an image to a website

### PR DESCRIPTION
**Pull request by rvs**
This makes HAWQ website much prettier and more hipster looking

### CODE REVIEW

**Comment by aayn** [comment]
Hey!

**Comment by GregChase** [comment]
I dun think this does what you think it does

File: index.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
 
     <div class="wrap">
 
-      <!-- <img src="images/logo-hawq-placeholder.jpg" alt=""/> -->
```
Comment (rvs) [inline index.html:78]: Fool! What have you done? You totally broke it!

**Comment by rvs** [comment]
Oh, sorry guys, I need to fix it.

**Comment by GregChase** [comment]
I can guarantee this is just as good as before.

**Pull request closed without merging**